src/main/kotlin/no/nav/familie/klage/behandling/dto/BehandlingDto.kt

```diff
@@ -17,10 +17,10 @@ data class BehandlingDto(
     val resultat: BehandlingResultat?,
     val opprettet: LocalDateTime,
     val vedtaksdato: LocalDateTime? = null,
-    val stønadstype: Stønadstype
+    val ytelsestype: Ytelsestype
```

Comment:
Jeg er usikker på ordbruket her, burde vi byte? 
Hvorfor har tilbakekreving brukt ytelsestype? 
Er det sånn att alle våre typer egentlige er stønadstyper?

Reply:
@ma10s @charliemidtlyng

Reply:
To ord, samme greia..?
https://data.nav.no/begrep/BEGREP-133 
https://data.nav.no/begrep/BEGREP-1130